actions/store_audio_info_MOD.js

```diff
@@ -201,7 +201,7 @@ module.exports = {
         result = Boolean(Audio.dispatchers[targetServer.id]); // is playing
         break;
       case 2:
-        result = this.dest(Audio.dispatchers[targetServer.id], 'startTime') || 0; // when the music first started playing
+        result = Audio.dispatchers[targetServer.id]?.startTime || 0; // when the music first started playing
```

Comment:
All the modifications to this file are irrelevant. It won't work either way

Reply:
should that file just be removed?

Reply:
Yes